ksp_plugin_adapter/flight_planner.cs

```diff
@@ -7,34 +7,254 @@
+              NavigationManoeuvre manoeuvre =
+                  plugin_.FlightPlanGetManoeuvre(vessel_guid, i);
+              if (manoeuvre.final_time > current_time) {
+                if (manoeuvre.burn.initial_time > current_time) {
+                  UnityEngine.GUILayout.TextArea("Upcoming manœuvre: #" +
+                                                 (i + 1));
+                  UnityEngine.GUILayout.Label(
+                      "Ignition " +
+                      FormatTimeSpan(TimeSpan.FromSeconds(
+                          current_time - manoeuvre.burn.initial_time)));
+                } else {
+                  UnityEngine.GUILayout.TextArea("Ongoing manœuvre: #" +
+                                                 (i + 1));
+                  UnityEngine.GUILayout.Label(
+                      "Cutoff " +
+                      FormatTimeSpan(TimeSpan.FromSeconds(
+                          current_time - manoeuvre.final_time)));
+                }
+                show_guidance_ =
+                    UnityEngine.GUILayout.Toggle(show_guidance_,
+                                                 "Show on navball");
+                if (show_guidance_ &&
+                    !double.IsNaN(manoeuvre.inertial_direction.x +
+                                  manoeuvre.inertial_direction.y +
+                                  manoeuvre.inertial_direction.z)) {
+                  if (guidance_node_ == null) {
+                    guidance_node_ = vessel_.patchedConicSolver.AddManeuverNode(
+                        manoeuvre.burn.initial_time);
+                  }
+                  Vector3d stock_velocity_at_node_time =
+                      vessel_.orbit.getOrbitalVelocityAtUT(
+                                       manoeuvre.burn.initial_time).xzy;
+                  Vector3d stock_displacement_from_parent_at_node_time =
+                      vessel_.orbit.getRelativePositionAtUT(
+                                       manoeuvre.burn.initial_time).xzy;
+                  UnityEngine.Quaternion stock_frenet_frame_to_world =
+                      UnityEngine.Quaternion.LookRotation(
+                          stock_velocity_at_node_time,
+                          Vector3d.Cross(
+                              stock_velocity_at_node_time,
+                              stock_displacement_from_parent_at_node_time));
+                  guidance_node_.OnGizmoUpdated(
+                      ((Vector3d)manoeuvre.burn.delta_v).magnitude *
+                          (Vector3d)(stock_frenet_frame_to_world.Inverse() *
+                                     (Vector3d)manoeuvre.inertial_direction),
+                      manoeuvre.burn.initial_time);
+                  should_clear_guidance = false;
+                }
+                break;
+              }
+            }
+            if (should_clear_guidance && guidance_node_ != null) {
+              vessel_.patchedConicSolver.RemoveManeuverNode(guidance_node_);
```

Comment:
In KSP it is; they generally go with american conventions: it's Kerbal Space *Program* rather than programme. To be honest they just tend to be wildly creative when it comes to spelling, e.g., `persistant` (is this Programme Spatial Kerbal?) or `gymbal`.